velox/common/base/Fs.h

```diff
@@ -15,10 +15,32 @@
  */
 #pragma once
 
+#include <optional>
+
 #if __has_include("filesystem")
 #include <filesystem>
 namespace fs = std::filesystem;
 #else
 #include <experimental/filesystem>
 namespace fs = std::experimental::filesystem;
 #endif
+
+namespace facebook::velox::common {
+
+/// Generates a file directory specified by 'dirPath'. The generation will be
```

Comment:
Can you clarify in the comment that this is to create on local fs? And if it is for testing purpose only, consider move to velox/exec/tests/utils/ Thanks!
```
s/generateFileDirectory/createFileDirectory/
s/generateFilePath/createFile/
```